specs/error_state/ErrorOutOfGasDynamicMemoryExpansion.md

```diff
@@ -57,3 +57,7 @@ else:
 2. `2` call context lookups for `is_success` and `rw_counter_end_of_reversion`.
 
 And restore context lookups for non-root call.
+
```

Comment:
Under the two sections Constraints and Lookups only `CREATE` is mentioned (missing `CREATE2`).

Reply:
Nice catch! fixed in [0b8af97](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/467/commits/0b8af97c5b95860d35e13749d3333b56d04045bb)